app/services/foreman/renderer/scope/report.rb

```diff
@@ -121,7 +121,7 @@ def report_render_csv
         end
 
         def report_render_html
-          html = ""
+          html = +""
```

Comment:
Here I chose to be lazy and explicitly use a mutable string instead of changing the source.